TWLight/graphs/helpers.py

```diff
@@ -80,7 +89,11 @@ def get_data_count_by_month(queryset, data_format=JSON):
             else:
                 timestamp = current_date
 
-            num_objs = queryset.filter(date_created__lte=current_date).count()
+            try:
```

Comment:
Likewise here - it's quite opaque reading this why the `try` is necessary. Doing a check for whether we're looking at an `Authorization` queryset or an `Application` one would at a minimum help the readability of this code.